adapters/core2sn/class.go

```diff
@@ -59,7 +59,7 @@ func AdaptSierraClass(class *core.SierraClass) *starknet.SierraDefinition {
 	}
 }
 
-func AdaptCairo0Class(class *core.DeprecatedCairoClass) (*starknet.Cairo0Definition, error) {
+func AdaptDeprecatedCairoClass(class *core.DeprecatedCairoClass) (*starknet.Cairo0Definition, error) {
 	decompressedProgram, err := utils.Gzip64Decode(class.Program)
```

Comment:
Here, let's change `starknet.Cairo0Definition` to `DeprecatedCairoClass`. There won't be a name clash because they are in different packages